frontend/PLUGIN.md

```diff
@@ -35,47 +35,7 @@ npm run setup
 npm run plugins
 ```
 
-This will concurrently start the backend server and frontend webpack development server.
-
-### [OCP Console Git Repo](https://github.com/openshift/console)
-
-Ensure that [OCP Console's dependencies](https://github.com/openshift/console#dependencies) are met in your local environment.
-
-```
-git clone git@github.com:openshift/console.git
-```
-
-### Running OCP Console in development mode
```

Comment:
I think it is worth keeping this section to cover the case when it is necessary to run the plugins against development mode console e.g. when a change in console codebase is needed.

Reply:
Ok, I've updated the README to contain sections for both - running the plugins against OCP downloaded from quay and running against local development build